server/job.go

```diff
@@ -44,10 +44,13 @@ func (p *Plugin) runBulkDeleteJob(dryRun bool, runningUserId string, runningChan
 		return
 	}
 
-	bulkDelete(p.pluginClient, p.socketClient, statusPost, usersToDelete, func(status int) {
+	if success := bulkDelete(p.pluginClient, p.socketClient, statusPost, usersToDelete, func(status int) {
 		statusPost.Message = fmt.Sprintf("### Bulk user deletion job started\nDeleted %d/%d users...", status, userCount)
 		p.pluginClient.Post.UpdatePost(statusPost)
-	})
+	}); success {
```

Comment:
This will only be success if all the users are successfully deleted. For partial success will they still receive the list of successful deletions?

Reply:
I just created another task to address your concern here: #11

It will require a deeper change that I'd like to keep isolated. This PR will at least ensure we don't swallow any errors :-)